core/src/main/scala/sbt/librarymanagement/EvictionWarning.scala

```diff
@@ -195,7 +195,8 @@ final class EvictionWarning private[sbt] (
     val scalaEvictions: Seq[EvictionPair],
     val directEvictions: Seq[EvictionPair],
     val transitiveEvictions: Seq[EvictionPair],
-    val allEvictions: Seq[EvictionPair]
+    val allEvictions: Seq[EvictionPair],
+    val binaryIncompatibleEvictionExists: Boolean
```

Comment:
Oh, this line breaks binary compatibility....
I'm thinking how to fix #287 without binaryIncompatibleEvictionExists field now.